app/controllers/api/v1/search_controller.rb

```diff
@@ -612,6 +650,19 @@ def self.generate_bq_query_string(facets)
         with_statement + base_query + from_clause + " WHERE " + where_clauses.join(" AND ")
       end
 
+      # convert a list of facet filters into a keyword search for inferred matching
+      def self.convert_filters_for_inferred_search(facets:)
+        filter_terms = []
+        facets.each do |facet|
+          search_facet = SearchFacet.find(facet[:object_id])
+          # only use non-numeric facets
+          if !search_facet.is_numeric?
+            filter_terms += search_facet[:filters].map {|filter| filter[:name]}
```

Comment:
I think this can be rewritten as 
```suggestion
            filter_terms += search_facet.filters.map(&:name)
```

Reply:
Actually it cannot.  The shorthand block notation does not work with hash keys (I wish it did).